mitype/app.py

```diff
@@ -348,6 +372,8 @@ def update_state(self, win):
             win.addstr(" to see replay, ")
             win.addstr(" Tab ", curses.color_pair(7))
             win.addstr(" to retry ")
+            win.addstr(" CTRL+S ", curses.color_pair(7))
```

Comment:
While testing on Ubuntu, I noticed that the terminal freezes and doesn't open webbrowser. There's some mechanism in the terminal which freezes IO, which can then be unfreezed with ctrl+q.
Let's make this CTRL+T to tweet result, instead of share result.